src/ui/public/vis/request_handlers/courier.js

```diff
@@ -0,0 +1,51 @@
+import _ from 'lodash';
+
+const CourierRequestHandlerProvider = function (Private, courier, timefilter) {
+  return {
+    name: 'courier',
+    handler: function (vis, appState, uiState, searchSource) {
```

Comment:
courier will issue a request when filters, query, time or aggs change.

i am guessing that the zoom level changes somehting in the aggs or sets a filter ?